.github/ISSUE_TEMPLATE/ice.md

```diff
@@ -8,6 +8,11 @@ Thank you for finding an Internal Compiler Error! 🧊  If possible, try to prov
 a minimal verifiable example. You can read "Rust Bug Minimization Patterns" for
 how to create smaller examples.
 http://blog.pnkfx.org/blog/2019/11/18/rust-bug-minimization-patterns/
+
+Please check if your ICE has already been reported by searching for parts of the ICE-message
+e.g. "invalid asymmetric binary op Lt" or "assertion failed: self.let_source != LetSource::None"
+and the source location of the ICE, such as "compiler/rustc_const_eval/src/interpret/operator.rs"
+in the bugtracker and leave a comment with your reproducer if you find tickets corresponding to your ICE.
 -->
 
```

Comment:
Discussion: I wonder if we should have a quick checkbox TODO list for the reporter:

```markdown
- [ ] I searched for parts of the ICE message in the issue tracker, and could not find duplicates
- [ ] xxx
```

though it does introduce a little bit more friction.

Reply:
Probably not really useful.